### PR TITLE
Add EL9 links to the nightly upgrade guide

### PR DESCRIPTION
**Pull request by ianballou**
There may be more spots that need EL9.

### CODE REVIEW

File: guides/common/modules/proc_upgrading-a-connected-project-server.adoc [hunk 1]
```diff
@@ -83,7 +83,19 @@ ifdef::foreman-el,katello[]
 ----
 # {project-package-update}
 ----
-. Update repositories
+. Update repositories for EL9:
```
Comment (Lennonka) [inline guides/common/modules/proc_upgrading-a-connected-project-server.adoc:86]:
```suggestion
. Update repositories for {EL} 9:
```
Also elsewhere.

**Review by Lennonka — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_upgrading-a-connected-project-server.adoc (hunk 1 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_upgrading-a-connected-project-server.adoc:86]: Will it be better if we rephrase this to:

> For {EL} 9 users

and 

> For {EL} 8 users

@Lennonka

**Review by ekohl — CHANGES_REQUESTED** [review]
I would keep the whole procedures more separate. For example, `dnf -y module switch-to postgresql:13` doesn't work on EL9 because it ships PG 13 in the base. We also don't have the `foreman:el8` module on EL9.

**Comment by ianballou** [comment]
It felt a bit verbose to copy / paste the  upgrade bits and repeat them for EL8 and EL9, so I added some "if" conditionals for EL 8 users. If anyone has a suggestion for a neat way to separate them even more, let me know via a change request.

File: guides/common/modules/proc_upgrading-smartproxy-server.adoc [hunk 2]
```diff
@@ -62,13 +74,13 @@ ifdef::katello[]
 # dnf module disable pulpcore
 ----
 endif::[]
-. Switch to the PostgreSQL 13 module:
+. Switch to the PostgreSQL 13 module if running {EL} 8:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_upgrading-smartproxy-server.adoc:77]:
```suggestion
. On {EL} 8: Switch to the PostgreSQL 13 module:
```

same for below.

File: guides/common/modules/proc_upgrading-smartproxy-server.adoc [hunk 3]
```diff
@@ -62,13 +74,13 @@ ifdef::katello[]
 # dnf module disable pulpcore
 ----
 endif::[]
-. Switch to the PostgreSQL 13 module:
+. On {EL} 8: Switch to the PostgreSQL 13 module:
```
Comment (ekohl) [inline guides/common/modules/proc_upgrading-smartproxy-server.adoc:77]: Pre-existing problem, but this part should be inside `ifdef::katello[]` since PostgreSQL isn't on a Smart Proxy with vanilla Foreman. Only with Katello.

**Review by ekohl — APPROVED** [review]
In itself this is correct and improves it.

File: guides/common/modules/proc_upgrading-smartproxy-server.adoc (hunk 3 above)
Comment (ianballou) [reply guides/common/modules/proc_upgrading-smartproxy-server.adoc:76]: I moved the endif::[] to include this in the katello section.

File: guides/common/modules/proc_upgrading-a-connected-project-server.adoc [hunk 4]
```diff
@@ -83,7 +83,19 @@ ifdef::foreman-el,katello[]
...
+----
+ifdef::katello[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
+https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
+endif::[]
+ifdef::foreman-el[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
+endif::[]
+----
+. On {EL} 9: Update repositories:
```
[the remark this answers is not in the document]
Comment (ianballou) [reply guides/common/modules/proc_upgrading-a-connected-project-server.adoc:98]: ah good catch

File: guides/common/modules/proc_upgrading-a-connected-project-server.adoc [hunk 5]
```diff
@@ -83,7 +83,19 @@ ifdef::foreman-el,katello[]
...
+----
+ifdef::katello[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
+https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
+endif::[]
+ifdef::foreman-el[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
+endif::[]
+----
+. On {EL} 8: Update repositories:
```
Comment (Lennonka) [inline guides/common/modules/proc_upgrading-a-connected-project-server.adoc:98]:
```suggestion
. Update repositories:
.. On {EL} 9:
+
[options="nowrap" subs="attributes"]
----
ifdef::katello[]
# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
endif::[]
ifdef::foreman-el[]
# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
endif::[]
----
.. On {EL} 8:
```
This structure would be a bit less prone to oversights.

File: guides/common/modules/proc_upgrading-smartproxy-server.adoc [hunk 6]
```diff
@@ -42,7 +42,19 @@ endif::[]
...
+----
+ifdef::katello[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm \
+https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/x86_64/katello-repos-latest.rpm
+endif::[]
+ifdef::foreman-el[]
+# {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
+endif::[]
+----
+. On {EL} 8: Update repositories:
```
Comment (Lennonka) [inline guides/common/modules/proc_upgrading-smartproxy-server.adoc:57]: Similarly here.

**Review by Lennonka — CHANGES_REQUESTED** [review]
One more structural suggestion.

**Comment by apinnick** [comment]
@ianballou Can you please take a look at this?

**Review by Lennonka — APPROVED** [review]
Supergreen!

**Review by maximiliankolb — APPROVED** [review]
Thanks Ian, LGTM.
